Add line about CMCLIENT_OPTS variable (#29)
SAP/devops-cm-client: README.md

<CMD> [COMMON_OPTIONS...] <subcommand> [SUBCOMMAND_OPTIONS] <parameters...>
````

> To pass additional Java options to the command e.g. another truststore you can set the environment variable `CMCLIENT_OPTS`

Reviewer: ```suggestion
To pass additional Java options to the command (for example, another truststore), set the environment variable `CMCLIENT_OPTS`
```
